README.md

```diff
@@ -6,12 +6,12 @@ INSTALL
 =======
 - Python3 is required
 - Make the following changes before build and install:
-    - enable `GTSAM_INSTALL_CYTHON_TOOLBOX`
-    - if you want to build the gtsam python library for a specific python version (eg 3.6), use the `-DGTSAM_PYTHON_VERSION=3.6` option, otherwise the default interpreter will be used.
+    - Set the CMake flag `GTSAM_INSTALL_CYTHON_TOOLBOX=ON` to enable building the cython wrapper.
```

Comment:
```
- Set the CMake flag `GTSAM_INSTALL_CYTHON_TOOLBOX` to `ON` to enable building the cython wrapper.
- Set the CMake flag `GTSAM_PYTHON_VERSION` to `3`, otherwise the default interpreter will be used.
You can do this on the command line as follows:
```